Cargo.toml

```diff
@@ -13,6 +13,7 @@ members = [
   "common/network-types",
   "common/parallelize",
   "common/primitive-types",
+  "common/traits",
```

Comment:
Why could it not go under `hopr-lib/api`? This is, after all, api interface.

Reply:
I have to use it in `hopr-db` and `hopr-lib` later depends on `hopr-db`.